nightfall-client/src/services/kem-dem.mjs

```diff
@@ -105,19 +105,50 @@ const deDem = (encryptionKey, ciphertexts) => {
   return plainTexts;
 };
 
+/**
+This function generates the transfer key pair for the observers
+@function genTransferKeysForObservers
+@param {GeneralNumber} senderPrivateKey - The private key of the sender
+@param {Array<GeneralNumber>} receiverPublicKey - The public pkd of the recipient
+@returns {Promise<Array<GeneralNumber, Array<BigInt>>>} The private and public key pair
+*/
+const genTransferKeysForObservers = async (senderPrivateKey, receiverPublicKey) => {
```

Comment:
You mention here `Observers`. Who are they? It would be good to have some details in the jsdoc or even add them to a README.

Reply:
Done. Added to the description of the jsdoc. Observers are receiver and regulator.